barretenberg/cpp/src/barretenberg/vm2/tracegen_helper.cpp

```diff
@@ -99,6 +99,11 @@ auto build_precomputed_columns_jobs(TraceContainer& trace)
             AVM_TRACK_TIME("tracegen/precomputed/to_radix_p_decompositions",
                            precomputed_builder.process_to_radix_p_decompositions(trace));
         },
+        [&]() {
+            // public_inputs.sel is precomputed. Should it be populated by the precomputed builder?
```

Comment:
I think this makes sense. You can remove the comment.

Reply:
Ahh, I see, you mean instead of the `PublicInputsTraceBuilder`... I don't have a preference! but it would make sense for consistency.